src/classes/RoomListings.js

```diff
@@ -1,6 +1,6 @@
 class RoomListings {
-  constructor() {
-
+  constructor(rooms) {
+    this.hotelRooms = rooms;
   }
```

Comment:
I would add all of the rooms data keys and values in this constructor so you can write a test validating that all that information has been inputted for all rooms.

Reply:
Not sure if that's 100% necessary, but that's how I did it.